### PR TITLE
Fix Multiple Alerts

### LINKED ISSUE #253: Multiple alerts in one message all trigger separate Pax DMs
For example if you have the alert `hello` and the alert `world`, the sentence `hello world` would send you two DMs

### PR DESCRIPTION
**Pull request by danparizher**
Fixes #253 and also adds self typing across the project. Reviewers should look at `alerts.py`

### CODE REVIEW

File: cogs/alerts.py [hunk 1]
```diff
@@ -301,9 +302,24 @@ async def user_alerts() -> None:
                 if not message.channel.permissions_for(member).view_channel:
                     continue
 
+                if uid not in alerts:
+                    alerts[uid] = []
```
Comment (GabeMillikan) [inline cogs/alerts.py:306]: there is a builtin dictionary function for this: `alerts.setdefault(uid, [])`

**Review by GabeMillikan — APPROVED** [review]
very elegant fix, nice

**Review by sebastiaan-daniels — APPROVED** [review]
very gud

**Comment by sebastiaan-daniels** [comment]
As described in [5.2.3 of the Contributing guidelines](https://github.com/danparizher/Pax-Academia/blob/main/CONTRIBUTING.md#523-reviewing), only two reviews are needed for a merge.

**Pull request merged**